wiki/fr/fr-Installation-for-Macintosh.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: wiki
+title: "Installation sous MacOS"
+lang: "fr"
+permalink: "/wiki/Installation-for-Macintosh"
+---
+
+# Installation sous MacOS
+
+Assurez-vous de lire le [guide de démarrage et installation](Getting-Started) avant de commencer.
+
+1. [Téléchargez Jamulus](https://sourceforge.net/projects/llcon/files/latest/download){: target="_blank" rel="noopener noreferrer" .button}
+1. **Décompressez le fichier .zip téléchargé**. Double-cliquez sur le fichier `.zip`, ce qui créera un nouveau dossier du même nom. Le dossier contient le fichier de licence et un fichier `.dmg` contenant l'application.
+1. **Double-cliquez sur le fichier `.dmg`** pour l'ouvrir. L'image s'ouvre et vous verrez deux applications (client et serveur Jamulus).
+1. **Glissez et déposez les deux icônes dans le dossier Applications** pour installer Jamulus.
+1. **Executez Jamulus**. Vous devriez maintenant pouvoir utiliser Jamulus comme n'importe quelle autre application. Si vous voyez un message qui vous dit que Jamulus ne peut pas être ouvert parce que le développeur ne peut pas être vérifié, lisez la suite et jetez un coup d'oeil à la [section suivante](#jamulus-ne-peut-pas-être-ouvert-car-le-développeur-ne-peut-pas-être-vérifié).
+
+_Vous pouvez supprimer le dossier dans le répertoire Downloads contenant le `.dmg` et éjecter le lecteur "Jamulus" de votre bureau. Ils ne sont plus nécessaires._
+
+Vous pouvez également [compiler vous même le logiciel](Compiling) si vous en avez envie.
+
+## "Jamulus" ne peut pas être ouvert car le développeur ne peut pas être vérifié
+
+**La première fois que vous essaierez de double-cliquer sur Jamulus pour l'exécuter, vous verrez probablement un message disant qu'il ne peut pas être ouvert.**
+
+Pour résoudre ce problème, accédez au dossier Applications via le Finder, cliquez sur Jamulus, faites un clic droit (ou un contrôle-clic) et sélectionnez « _Ouvrir_ » dans le haut du menu contextuel.
+
+Vous aurrez alors une version légèrement différente du même message, qui vous permettra de cliquer sur « _Ouvrir_ » et de continuer à l'exécuter. Cet avertissement ne s'affichera plus - il suffit de double-cliquer dessus pour le lancer. Pour plus d'informations, consultez [cette page d'assistance Apple](https://support.apple.com/fr-fr/guide/mac-help/mh40616/mac){: target="_blank" rel="noopener noreferrer"}.
+
```

Comment:
* typo: "aurrez" -> "aurez"
* "« _Ouvrir_ »" I can't find such a layout on the [English version](https://raw.githubusercontent.com/jamulussoftware/jamuluswebsite/changes/wiki/en/en-Installation-for-Macintosh.md). Please, don't add it in the French translation if that's not in the English version first.
* "_il suffit de double-cliquer dessus pour le lancer_" -> "_il suffira de double-cliquer dessus pour le lancer_"

Reply:
"_« Ouvrir »_": still there is not such "_«_" & "_»_" in the English version where it's simply written "_which allows you to click Open and go ahead with running_"

Reply:
> there is not such "_«_" & "_»_" in the English version where it's simply written "_which allows you to click Open and go ahead with running_"

There is in the paragraph before, maybe it has been forgotten here.

Reply:
Then, once again, it has to be upstreamed in the English version first.
Then, if/when accepted, that will go down to the translations and translators on the next update.
Please, remove it.